source/blog/2016-01-19-atomic-app-0-4-0.html.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Atomic App 0.4.0 release!
+author: cdrage
+date: 2016-01-21 19:20:00 UTC
+tags: atomic app, nulecule, composite apps, developers, alm, application lifecycle
+comments: true
+published: false
+---
+
+A fast release since 0.3.1, we've now released [0.4.0](https://github.com/projectatomic/atomicapp/releases) of [Atomic App](http://www.projectatomic.io/docs/atomicapp/)!
+
+Our biggest change is the renaming of `atomicapp install` to `atomicapp fetch`.
+
+An example of fetching a packaged Atomic App container is:
+```bash
+  atomicapp fetch projectatomic/helloapache --destination ~/
```

Comment:
Don't use `~/`, use maybe `./` instead?

add the atomic equivalent command here as well.
`atomic run projectatomic/helloapache --mode=fetch --destination ./`